doc/external-libs/trilinos.html

```diff
@@ -116,6 +116,27 @@
 
     <h5>BLAS and LAPACK</h5>
 
+    <p style="color: red">
+      Note: At the time of writing (November 2016) Trilinos cannot link against 
+      LAPACK 3.6.0 or later, due to two symbols that were deprecated in LAPACK 
+      3.6.0. To fix this, edit the Trilinos file 
```

Comment:
Do you mean that these were actually *removed*? Deprecated would still mean that they're there, but that their use is not recommended any more.